doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -6,6 +6,11 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version 7.7.x to 7.8, include enhancements, security updates, and bug fixes.
 
+[IMPORTANT]
+====
+{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. DMN 1.1 and DMN 1.3 models are not currently supported in the DMN designer in {CENTRAL}.
```

Comment:
I ASSuME that you are hard-coding 7.9 on purpose for this particular note. If not  - Fix it LOL

Reply:
Yes, you assumed right :) This is going into the 7.8 docs and we don't have a future version attribute. (Maybe we should have past and future version attributes? WDYT?)